### DISCUSSION TITLE
How to update translations

### DISCUSSION THREAD

**Discussion by vinod-aot** [AOT-Technologies/forms-flow-ai#1191] [Q&A]
The translations used as part of Internationalization need some correction. They are literal translations of the English word, which may not be appropriate for the context in the specific language.

Is it possible to change the translations, or is it dynamic translations?

**Comment by Bonymol-aot** [comment]
@vinod-aot  Currently we are providing word by word  translations, which is not dynamic. If you need to update any  existing translation, you can easily update it by changing the value against the particular key inside the required resourceBundles.js file located at https://github.com/AOT-Technologies/forms-flow-ai/tree/master/forms-flow-web/src/resourceBundles

**Comment by sreehari-aot** [comment]
@vinod-aot It is not dynamic translation meaning we are not relying on any external API to perform the translations, but it is not completely word-by-word translation either. In https://github.com/AOT-Technologies/forms-flow-ai/tree/master/forms-flow-web/src/resourceBundles you can add the sentence and its meaningful translations of different languages. But it is not as effective as a dynamic translation because every time we need to keep the translations up to date.
